packages/docs-page/style.module.css

```diff
@@ -0,0 +1,132 @@
+@custom-media --mobile-viewports (max-width: 939px);
+@custom-media --desktop-viewports (min-width: 940px);
+
+.root {
+  & :global(.jump-to-section) {
```

Comment:
Changing this would require a refactor of the `jump-to-section` functionality, which given the aim of supporting `global-styles` changes only felt outside the scope of this PR.